eng/versioning/external_dependencies.txt

```diff
@@ -67,8 +67,8 @@ io.netty:netty-tcnative-boringssl-static;2.0.62.Final
 io.netty:netty-transport-native-epoll;4.1.101.Final
 io.netty:netty-transport-native-unix-common;4.1.101.Final
 io.netty:netty-transport-native-kqueue;4.1.101.Final
-io.projectreactor.netty:reactor-netty;1.0.39
-io.projectreactor.netty:reactor-netty-http;1.0.39
+io.projectreactor.netty:reactor-netty;1.0.40
+io.projectreactor.netty:reactor-netty-http;1.0.40
```

Comment:
Do we want to update these versions before release? Are there any live test results that can validate that this new version of reactor-netty* doesn't break anything?

Reply:
We should, we've upgraded Netty and Reactor dependencies which are effectively aligned with Reactor Netty 1.0.40 but it looks like we forgot to upgrade it from 1.0.39